cg/store/api/findbusinessdata.py

```diff
@@ -166,6 +166,31 @@ def family_samples(self, family_id: str) -> List[models.FamilySample]:
             .all()
         )
 
+    def sample_cases(self, sample_id: str) -> List[models.FamilySample]:
+        """Find the cases associated with a sample."""
+        return (
+            self.FamilySample.query.join(models.FamilySample.family, models.FamilySample.sample)
+            .filter(models.Sample.internal_id == sample_id)

```

Comment:
I think we should split the query and filter here to make it less complex and reusable